packages/desktop/src/renderer/app/components/environment-logs/environment-logs.component.html

```diff
@@ -115,8 +115,18 @@
                     </div>
                   </div>
                 </div>
-                <div class="nav-link-subtitle ellipsis">
-                  {{ log.timestamp | date : 'yyyy-MM-dd HH:mm:ss:SSS' }}
+                <div class="d-flex align-items-center">
+                  <div class="d-flex nav-link-subtitle ellipsis">
+                    {{ log.timestamp | date : 'yyyy-MM-dd HH:mm:ss:SSS' }}
+                  </div>
+                  <div class="d-flex ml-auto ellipsis nav-link-subtitle">
+                    <span
+                      class="mt-0 badge badge-default ml-0 response-badge-{{
```

Comment:
Remove ml-0 to ensure there is a gap between the truncated (ellipsis) date and the badge

```suggestion
                      class="mt-0 badge badge-default response-badge-{{
```